pkgs/applications/networking/cluster/kubernetes/default.nix

```diff
@@ -1,55 +1,67 @@
-{ stdenv, fetchFromGitHub, which, go, makeWrapper, iptables, rsync, utillinux, coreutils, e2fsprogs, procps-ng }:
+{ stdenv, lib, fetchFromGitHub, which, go, makeWrapper, rsync
+, iptables, coreutils
+, components ? [
+    "cmd/kubectl"
+    "cmd/hyperkube"
+    "cmd/kube-dns"
+  ]
+}:
+
+with lib;
 
 stdenv.mkDerivation rec {
   name = "kubernetes-${version}";
-  version = "1.2.4";
+  version = "1.3.5";
```

Comment:
@offlinehacker I don't know if you noticed but the latest release is now `v1.4.0` (https://github.com/kubernetes/kubernetes/releases/tag/v1.4.0).